src/imgui/ImGuiMedia.cc

```diff
@@ -1501,6 +1503,14 @@ void ImGuiMedia::cassetteMenu(const TclObject& cmdResult)
 			if (auto* mute = dynamic_cast<BooleanSetting*>(controller.findSetting("cassetteplayer_ch1_mute"))) {
 				Checkbox(hotKey, "Mute tape audio", *mute, [](const Setting&) { return std::string{}; });
 			}
+			auto& player = reactor.getMotherBoard()->getCassettePort().getPluggedCasDev();
+			bool enabled = player.isMotorControlEnabled();
+			bool changed = ImGui::Checkbox("Motor control enabled", &enabled);
+			if (changed) {
+				manager.execute(TclObject(strCat("cassetteplayer motorcontrol ", enabled ? "on" : "off")));
```

Comment:
Detail. Here strCat() constructs a string, but one of the first things that the Tcl interpreter does with this string is to split it into a list of tokens. It's more efficient, and more important also easier, to directly create a Tcl list. Something like:
```
manager.execute(makeTclList("cassetteplayer", "motorcontrol", enabled ? "on" : "off"));
```

Reply:
OK, addressed this in a new push.